### ISSUE TITLE
ClosingHeredocIndentation should have an option to allow left rail alignment

### ISSUE THREAD

**Issue by pt-stripe** [rubocop-hq/rubocop#6016]
Many of our developers don't internalize the difference between `<<~` and `<<-`. I'd like to allow:

```
def method
  foo = <<~END
    bar
END
end
```

--------

## Expected behavior

This is allowed

## Actual behavior

This is autocorrected to indent.

## Steps to reproduce the problem

Use my source file with this cop enabled.

## RuboCop version

```
$ rubocop -V
0.57.2 (using Parser 2.5.1.0, running on ruby 2.4.1 x86_64-darwin16)
```

**Comment by koic** [comment]
That is the behavior that this `Layout/ClosingHeredocIndentation` cop expected.
If `<< ~` is used, it should be written as follows.

```ruby
def method
  foo = <<~END
    bar
  END
end
```

If it is difficult to write like that, you can disable this cop.

This cop will lose essence if this cop allow the following style of writing.

```ruby
def method
  foo = <<~END
    bar
END
end
```

So I think that it is not preferable.

**Comment by pt-stripe** [comment]
Ok. Understandable. Thanks for your consideration.

**Comment by koic** [comment]
Thanks too. I will close this issue, but please your feedback if there is something again.
